packages/atomic/src/components/common/item-link/item-link.spec.ts

```diff
@@ -0,0 +1,432 @@
+        target: '_blank',
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toHaveAttribute('target', '_blank');
+    });
+
+    it('should default to _self target when not provided', async () => {
+      const element = await renderComponent({});
+
+      const link = element.querySelector('a');
+      expect(link).toHaveAttribute('target', '_self');
+    });
+
+    it('should apply tabIndex when provided', async () => {
+      const element = await renderComponent({
+        tabIndex: 5,
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toHaveAttribute('tabindex', '5');
+    });
+
+    it('should not apply tabindex when not provided', async () => {
+      const element = await renderComponent({});
+
+      const link = element.querySelector('a');
+      expect(link).not.toHaveAttribute('tabindex');
+    });
+
+    it('should apply aria-hidden when ariaHidden is true', async () => {
+      const element = await renderComponent({
+        ariaHidden: true,
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toHaveAttribute('aria-hidden', 'true');
+    });
+
+    it('should not apply aria-hidden when ariaHidden is false', async () => {
+      const element = await renderComponent({
+        ariaHidden: false,
+      });
+
+      const link = element.querySelector('a');
+      expect(link).not.toHaveAttribute('aria-hidden');
+    });
+  });
+
+  describe('custom attributes', () => {
+    it('should apply custom attributes when provided', async () => {
+      const customAttributes = [
+        {nodeName: 'data-test', nodeValue: 'test-value'} as Attr,
+        {nodeName: 'data-id', nodeValue: '123'} as Attr,
+      ];
+
+      const element = await renderComponent({
+        attributes: customAttributes,
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toHaveAttribute('data-test', 'test-value');
+      expect(link).toHaveAttribute('data-id', '123');
+    });
+
+    it('should handle empty attributes array', async () => {
+      const element = await renderComponent({
+        attributes: [],
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toBeInTheDocument();
+    });
+
+    it('should handle undefined attributes', async () => {
+      const element = await renderComponent({
+        attributes: undefined,
+      });
+
+      const link = element.querySelector('a');
+      expect(link).toBeInTheDocument();
+    });
+  });
+
+  describe('event handlers', () => {
+    it('should call onMouseOver when mouse enters', async () => {
+      const handleMouseOver = vi.fn();
+      const element = await renderComponent({
+        onMouseOver: handleMouseOver,
+      });
+
+      const link = element.querySelector('a')!;
+      const mouseOverEvent = new MouseEvent('mouseover', {bubbles: true});
+      link.dispatchEvent(mouseOverEvent);
+
+      expect(handleMouseOver).toHaveBeenCalledTimes(1);
```

Comment:
```suggestion
      expect(handleMouseOver).toHaveBeenCalledOnce();
```

you can replace all "toHaveBeenCalledTimes(1)" instances.